.vscode/tasks.json

```diff
@@ -125,13 +163,21 @@
       "id": "verboseOutput",
       "type": "pickString",
       "description": "Output verbose logging? Select -v for verbose, and blank for quiet.",
-      "options": ["", "-v"],
+      "options": [
+        "",
+        "-v"
+      ],
       "default": "-v"
     },
     {
       "id": "releaseOverride",
       "type": "promptString",
       "description": "Release Version: Leave blank to grab the latest release, or provide the release manually (Ex 49.11.0)"
+    },
+    {
+      "id": "releaseBranch",
```

Comment:
Future Improvement: Merge this argument and the one above together. Eventually I want to be able to grab the latest release branch like we do for the change log generator when a user doesn't explicitly specify it.